nodestream/pipeline/extractors/files.py

```diff
@@ -66,6 +71,19 @@ def read_file_from_handle(self, fp: StringIO) -> Iterable[JsonLikeDocument]:
         return (json.loads(line) for line in fp)
 
 
+class ParquetFileFormat(SupportedFileFormat, alias=".parquet"):
+    def read_file_from_handle(self, fp: StringIO) -> Iterable[JsonLikeDocument]:
+        df = pd.read_parquet(fp, engine='pyarrow')
+
+        # Note: 
+        # It is not efficient to do this this way, which is
+        # converting the data frame to string JSON representation
+        # and then calling json.loads to parse the string into a JSON object.
+        # Find a direct way to convert to JSON object from parquet.
+
+        return json.loads(df.to_json(orient ='records'))
```

Comment:
Is it possible to implement a basic test for this? 

There is a pattern established in [here](https://github.com/nodestream-proj/nodestream/blob/main/tests/unit/pipeline/extractors/test_files.py).